src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2872,21 +2871,31 @@ namespace WaterToAirHeatPumpSimple {
                          .glycol->getSpecificHeat(state, state.dataSize->PlantSizData(PltSizNum).ExitTemp, RoutineNameAlt);
 
                 if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Heating) {
-                    RatedWaterVolFlowRateDes = simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
+                    RatedWaterVolFlowRateDes =
+                        (1 - 1 / RatedHeatCOP) * simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
```

Comment:
I do not think this is correct. Since when does Q = M * Cp * deltaT include COP? You have a load and a plant delta T, from those you calculate mass flow rate. I think this equation was correct to begin with. Also for the equation below for the companion coil, don't use (1 + 1/COP) but do use the correct plant Delta T from the Sizing:Plant object for the cooling plant. I believe PltSizNum here is for the heating plant. @Nigusse ?

Reply:
COP comes into play because as @Nigusse pointed out above, for the heating coil we have `m_dot_source = Q_load_side * (1 - 1/cop_h) / (cp_source * deltaT_source)`. Where `simpleWatertoAirHP.RatedCapHeat` is the load side capacity.

These are water to air coils, so the plant loop associated with them _should be_ a condenser plant loop, meaning that both the heating and cooling coil _should be_ on the demand side of that plant loop. So unless I'm missing something, the design dT should be the same for both coils. One thing I could do, to be safe, is to get the sizing plant object for the companion coil and use it to determine the cooling coil water flow rate instead of using the same value for both. I'll make that change.

Reply:
If this is for source side sizing then this is fine. I see that RatedCapHeat is the air-side heating capacity on the load side. So these equations make sense for adjustment to the source side. Yes, use the correct Sizing:Plant information.